pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "url-shortener"
+version = "0.1.0"
+description = "To shorter links"
+authors = ["eduardovale76 <eduardovale76@gmail.com>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+Django = "^4.1"
+pytest = "^7.1.2"
```

Comment:
Pytest também deve estar no dev-dependencies, tudo que só for utilizado deve estar no dev-dependencies. Geralmente são coisas relacionadas a lint e a testes.

Reply:
Feito!